spec/index.html

```diff
@@ -1350,9 +1358,9 @@ <h3>Escape Sequences</h3>
       <thead>
         <tr>
           <th></th>
-          <th><a href="#numeric">numeric<br/>escapes</a></th>
-          <th><a href="#string">string<br/>escapes</a></th>
-          <th><a href="#reserved">reserved character<br/>escapes</a></th>
+          <th><a href="#dfn-numeric-escape-sequence">numeric<br/>escapes</a></th>
```

Comment:
Using data-lt rather than href would be cleaner and make the internal indexing of term usages consistent. `<a data-lt=“numeric escape sequence”>numeric<br/>escapes</a>`.

Reply:
Done throughout the document.